### DISCUSSION TITLE
Ubiquiti UniFi Security Gateway 3P (USG-3P)

### DISCUSSION THREAD

**Discussion by iqrasyafaat** [helmiau/HelmiWrt-OS#64] [Q&A] [answered]
Pengen pasang HelmiWrt di UniFi Security Gateway (USG)
Bisa tolong dibuatkan gak bang

https://openwrt.org/toh/ubiquiti/unifi_security_gateway_3p

Terima Kasih

**Comment by helmiau** [comment, marked as answer]
tidak bisa, karena:
- arsitekturnya bukan aarch64, arm dan x86
- spek rendah

**Reply by iqrasyafaat** [reply]
Ok siap.. Masih lebih bagus pake Raspberry Pi 4 berarti ya bang?
Makasih infonya bang 😎👍

⭐⭐⭐⭐⭐

**Reply by helmiau** [reply]
kalo raspi 4 memang memang bagus, tapi sesuaikan kebutuhan saja bang.
